analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -205,15 +205,11 @@ class SpdxDocumentFile(
     }
 
     /**
-     * Return null in case of "NONE" or "NOASSERTION" to fit with the standard process ( concluded manage by curations )
+     * Return the concluded license to be used in ORT's data model, which expects a not present value to be null
+     * instead of NONE or NOASSERTION
      */
-    private fun buildConcludedLicense(pkg: SpdxPackage): SpdxExpression? {
-        return when {
-            pkg.licenseConcluded == SpdxConstants.NONE -> null
-            pkg.licenseConcluded == SpdxConstants.NOASSERTION -> null
-            else -> pkg.licenseConcluded.toSpdx()
-        }
-    }
+    private fun buildConcludedLicense(pkg: SpdxPackage): SpdxExpression? =
+        pkg.licenseConcluded.takeIf { SpdxConstants.isPresent(it) }?.toSpdx()
```

Comment:
If you want to keep this one-liner as a function instead of inlining it, I'd propose a name that does not use "build". The concluded license is not really built up / generated here, but just taken, with some condition. I'd just call this `getConcludeedLicense()`.